src/lib/exec.rs

```diff
@@ -441,7 +441,7 @@ impl Executor for Interpreter {
 
 impl Interpreter {
     /// https://tc39.es/ecma262/#sec-call
-    fn call(&mut self, f: &Value, v: &Value, arguments_list: Vec<Value>) -> ResultValue {
+    pub(crate) fn call(&mut self, f: &Value, v: &Value, arguments_list: Vec<Value>) -> ResultValue {
```

Comment:
What's this change?

Reply:
`find` method takes a callback to be applied on each element on the array. To call the function, we need access to this function in `array.rs`. So, I made it public in the crate. Usage [here](https://github.com/jasonwilliams/boa/pull/156/files#diff-c364bd20ac819c860ea950e0d684c3f0R276).

Reply:
Ok not seen that syntax before pub(crate)